workflows/bacterial_genomics/bacterial_genome_annotation/bacterial_genome_annotation-tests.yml

```diff
@@ -8,54 +8,54 @@
     Select a AMRFinderPlus database: amrfinderplus_V3.12_2024-05-02.2
   outputs:
     integronfinder2_logfile_text:
-      assert:
+      asserts:
         has_text:
           text: "Writing out results for replicon"
     integronfinder2_summary:
-      assert:
-        has_n_columns:
-          n: 6
+      asserts:
+        has_text:
+          text: "contig00001"
     integronfinder2_results_tabular:
-      assert:
-        has_n_columns:
-          n: 14
+      asserts:
+        has_text:
+          text: "contig00009_42"
     bakta_hypothetical_tabular:
-      assert:
-        has_n_columns:
-          n: 9
+      asserts:
+        has_text:
+          text: "DHJLLP_04750"
     bakta_annotation_json:
-      assert:
+      asserts:
         has_text:
           text: "aa_hexdigest"
     bakta_annotation_tabular:
-      assert:
-        has_n_columns:
-          n: 9
+      asserts:
+        has_text:
+          text: "Phosphotransferase system cellobiose-specific component IIC"
     isescan_results_tabular:
-      assert:
-        has_n_columns:
-          n: 24
+      asserts:
+        has_text:
+          text: "IS256_162"
     isescan_summary_tabular:
-      assert:
+      asserts:
         has_text:
           text: "nIS"
     isescan_logfile_text:
-      assert:
+      asserts:
         has_text:
           text: "Both complete and partial IS elements are reported."
     plasmidfinder_result_json:
-      assert:
+      asserts:
         has_text:
           text: "positions_in_contig"
     plasmidfinder_results_tabular:
-      assert:
+      asserts:
         has_n_columns:
           n: 8
     tooldistillator_summarize:
-      assert:
+      asserts:
         has_text:
           text: "CDS12738(DOp1)"
         has_text:
           text: "CALIN"
         has_text:
-          text: "insertion_sequence"
+          text: "PFAM"
```

Comment:
This will only test the last one, if you want to test all you need to write:
```
        - that: "has_text"
          text: "CDS12738(DOp1)"
        - that: "has_text"
          text: "CALIN"
        - that: "has_text"
          text: "PFAM"
```